src/cards/poison.ts

```diff
@@ -39,7 +39,6 @@ const spell: Spell = {
     },
   },
   modifiers: {
-    stage: 'Amount Flat',
```

Comment:
No stage for poison?

Reply:
I don't think so. There aren't any other onTurnEnd modifiers it can't overlap with in the unstaged group. Amount X is more for modifying a return value anyway (armor, damage/healing effectiveness, potion power, etc.)